cmd/gen.go

```diff
@@ -26,7 +26,7 @@ var genCmd = &cli.Command{
 			}
 		} else {
 			cfg, err = config.LoadConfigFromDefaultLocations()
-			if os.IsNotExist(errors.Cause(err)) {
+			if os.IsNotExist(errors.Unwrap(err)) {
```

Comment:
I think this simplifies to:
```suggestion
			if errors.Is(err, fs.ErrNotExist) {
```